sqlglot/dialects/doris.py

```diff
@@ -9,6 +9,21 @@
     time_format,
 )
 from sqlglot.dialects.mysql import MySQL
+from sqlglot.dialects.tsql import DATE_DELTA_INTERVAL
+
+
+def handle_date_trunc(self, expression: exp.DateTrunc) -> str:
+    expr = self.sql(expression, "this")
+    unit = expression.text("unit").lower()
+    if unit == "none":
+        return f"DATE({expr})"
+    else:
+        mapped_unit = (
```

Comment:
this is not very good code, can you do this properly?

Reply:
i'm not sure what this code is supposed to do, and i don't even think it's tested

Reply:
sorry, here is my problem